pyproject.toml

```diff
@@ -7,15 +7,12 @@ skip_magic_trailing_comma = true
 # for just these files, but doesn't seem possible yet.
 force-exclude = ".*_pb2.pyi"
 
-[tool.isort]
-profile = "black"
-combine_as_imports = true
-line_length = 130
-skip = [".git", ".github", ".venv"]
-extra_standard_library = [
-    "typing_extensions",
+[tool.ruff.isort]
+combine-as-imports = true
```

Comment:
I recommend also `split-on-trailing-comma = false`, for closer isort-like behavior.

Otherwise Ruff doesn't collapse multiline imports (even if they do fit on one line).

* See https://github.com/astral-sh/ruff/issues/6471